main.go

```diff
@@ -32,9 +35,21 @@ func mainInner() int {
 		opts.StartService = true
 	}
 
+	if clearCmds {
+		kbc, err := kbchat.Start(opts)
+		if err != nil {
+			fmt.Printf("error clearing advertisemnts: %v\n", err)
+			return 1
+		}
+		if err := kbc.ClearCommands(); err != nil {
+			fmt.Printf("error clearing advertisemnts: %v\n", err)
```

Comment:
Maybe this should return 1 or use a log.Critical. I also think the errors might go to stdout instead of stderr when using fmt.Println, but it isn't too big of a deal.

Reply:
switched over to log.Fatalf